.github/workflows/build.yml

```diff
@@ -339,3 +339,31 @@ jobs:
           RCLONE_CONFIG_PASS: ${{ secrets.RCLONE_CONFIG_PASS }}
         # Upload artifacts if not a PR && not a fork
         if: github.head_ref == '' && github.repository == 'rclone/rclone'
+
+  ios:
+    if: ${{ github.repository == 'rclone/rclone' || github.event.inputs.manual }}
+    timeout-minutes: 30
+    name: "ios-all"
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.16
+
+      - name: install gomobile
+        run: |
+          go get golang.org/x/mobile/cmd/gobind
+          go get golang.org/x/mobile/cmd/gomobile
+          env PATH=$PATH:~/go/bin gomobile init
+
+      - name: ios build
+        shell: bash
+        run: |
+          env PATH=$PATH:~/go/bin gomobile bind -v -target=iossimulator ./librclone/gomobile
```

Comment:
Does this produce a binary that is useful for something? If so then we could store it like we store the android builds, eg in https://beta.rclone.org/v1.58.0-beta.5947.b783f09fc/testbuilds/ ?

Reply:
IMO, we need more APIs to make rclone fully working on iOS because of the default config path is not writable.

I made a demo here: <https://github.com/imWildCat/rclone/commit/d37d7c8d8060aa265c62d12a899921a341b8b2fd#diff-8a1f1680ba70cb9a9b7d7c681da951adff5e19b1159ab09dd6b3d950a87dbfceR17-R19>

For this PR, I'd like to stay focused on make rclone buildable using gomobile.